OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - tallclair
+  - kubernetes/enhancements-admins
```

Comment:
This will need an OWNERS_ALIAS file and the enhancement-admins need to be re-defined there. Example - https://github.com/kubernetes/kubernetes-template-project/blob/master/OWNERS_ALIASES#L8

OWNERS file don't support GitHub teams currently :upside_down_face:

Reply:
Done :+1: